discretize/tests.py

```diff
@@ -633,6 +637,121 @@ def Quadratic(x, return_g=True, return_H=True):
     return Quadratic
 
 
+def assert_isadjoint(
+    forward,
+    adjoint,
+    shape_u,
+    shape_v,
+    complex_u=False,
+    complex_v=False,
+    clinear=True,
+    rtol=1e-6,
+    atol=0.0,
+    assert_error=True,
+):
+    r"""Dot product test for the forward operator and its adjoint operator.
+
+    Dot product test to verify the correctness of the adjoint operator
+    :math:`F^H` of the forward operator :math:`F`.
+
+    .. math::
+
+        \mathbf{v}^H ( \mathbf{F} \mathbf{u} ) =
+        ( \mathbf{F}^H \mathbf{v} )^H \mathbf{u}
+
+
+    Parameters
+    ----------
+    forward : function
```

Comment:
Forward and adjoint should be `callable` not function.

Reply:
Yes! I knew `function` was wrong, but I couldn't remember what it should be. Thanks.